otj-metrics-actuate/src/main/java/com/opentable/metrics/actuate/health/CodahaleHealthIndicatorConfiguration.java

```diff
@@ -0,0 +1,81 @@
+package com.opentable.metrics.actuate.health;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import com.codahale.metrics.health.HealthCheck;
+import com.codahale.metrics.health.HealthCheck.Result;
+
+import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
+import org.springframework.boot.actuate.autoconfigure.health.CompositeHealthIndicatorConfiguration;
+import org.springframework.boot.actuate.autoconfigure.health.ConditionalOnEnabledHealthIndicator;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.Health.Builder;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.boot.actuate.health.Status;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.context.annotation.Conditional;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+import io.micrometer.core.lang.NonNull;
+
+/**
+ * Presents {@link HealthCheck} beans as a composite {@link HealthIndicator}.
+ *
+ */
+@Configuration
+@ConditionalOnEnabledHealthIndicator("drop-wizard")
```

Comment:
What causes this to be enabled?

Reply:
@adamzr basically obeys rules defined at: https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-endpoints.html

Reply:
Add these comments to the code in rewofddd fashion